### PR TITLE
hbase-store: use rand instead of timestamp for selecing connection randomly

### PR DESCRIPTION
**Pull request by c4pt0r**

### CODE REVIEW

**Comment by c4pt0r** [comment]
@ngaut PTAL

File: store/hbase/kv.go [hunk 1]
```diff
@@ -72,7 +74,7 @@ func (s *hbaseStore) Begin() (kv.Transaction, error) {
 	s.mu.Lock()
 	defer s.mu.Unlock()
 	// get random conn
-	hbaseCli := s.conns[time.Now().UnixNano()%hbaseConnPoolSize]
+	hbaseCli := s.conns[rand.Intn(hbaseConnPoolSize)]
```
Comment (ngaut) [inline store/hbase/kv.go]: Extract s.conns[rand.Intn(hbaseConnPoolSize)] to a method.

**Comment by ngaut** [comment]
LGTM

**Comment by c4pt0r** [comment]
@qiuyesuifeng PTAL

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**